src/response_template.rs

```diff
@@ -148,6 +151,21 @@ impl ResponseTemplate {
         self
     }
 
+    /// Set a raw response body. The mime type needs to be set because the
+    // raw body could be of any type.
```

Comment:
It would be good to have a doc example here, setting for example a JSON response using bytes.